doc/ceph-volume/lvm/activate.rst

```diff
@@ -3,18 +3,19 @@
 ``activate``
 ============
 
-Once :ref:`ceph-volume-lvm-prepare` is completed, and all the various steps
-that entails are done, the volume is ready to get "activated".
+After :ref:`ceph-volume-lvm-prepare` has completed its run, the volume can be
+activated. 
 
-This activation process enables a systemd unit that persists the OSD ID and its
-UUID (also called ``fsid`` in Ceph CLI tools), so that at boot time it can
-understand what OSD is enabled and needs to be mounted.
+The activation of the volume involves enabling a systemd unit that persists the
```

Comment:
Activating the volume involves

Also I'm a markup slut so I'd backtick systemd, OSD ID, and UUID

Reply:
Accepted and... uh, indulged.

Reply:
That style guide we keep talking about ;)

Reply:
I'll announce at the 2024 Cephalocon that we'll have a style guide by 2025.